app/content/analytics-toolbox-snowflake/overview/getting-access.md

```diff
@@ -121,45 +96,63 @@ USE DATABASE "<my database>";
 
 ![Setup on Snowflake Classic Web Interface](/img/analytics-toolbox-snowflake/install1.png)
 
-##### 2. Check the installed version
+##### 2. Create the functions and procedures
+
+Download the [modules script](https://storage.googleapis.com/carto-analytics-toolbox-core/snowflake/latest/sql/carto-analytics-toolbox-core-snowflake-modules.sql) into a local file.
 
-{{% bannerNote title="ATTENTION" type="warning" %}}
-If this is the first time installing the toolbox, skip this step.
+{{% bannerNote title="TIP" type="tip" %}}
+If you have a previously installed version of the Analytics Toolbox, you can check the installed version by running `SELECT carto.VERSION_CORE()` and then compare it with the version of the toolbox you are about to install by checking [this version file](https://storage.googleapis.com/carto-analytics-toolbox-core/snowflake/latest/version).
 {{%/ bannerNote %}}
 
-Download the [version file](https://storage.googleapis.com/carto-analytics-toolbox-core/snowflake/latest/version).
+To install the functions and procedures of the Analytics Toolbox in the `carto` schema of your database, execute the downloaded file `carto-analytics-toolbox-core-snowflake-modules.sql`. Please note that must execute this file's commands in the same Worksheet where you executed all the previous statements of this installation guide. 
 
-Compare with your version installed: (execute this in the same session as the previous USE ROLE etc.)
+You can load the script into a Worksheet by using the dropdown menu on the top right and choosing "Load Script". The "All Queries" check seems to work unreliably with large scripts, so we advice to select all the text instead (just press Control-A or Command-A if using a Mac), then press the "Run" button and confirm you want to execute all the lines.
+
+![Setup on Snowflake Classic Web Interface](/img/analytics-toolbox-snowflake/install2.png)
+
+{{% bannerNote title="WARNING" type="warning" %}}
+This script will remove all the previously installed functions and procedures of the Analytics Toolbox in the `carto` schema.
+{{%/ bannerNote %}}
 
-```sql
-SELECT carto.VERSION_CORE();
-```
 
-You can also check the installed functions and procedures with:
+##### 3. Check installation
+
+You can check the installed functions and procedures by running the following statements:
 
 ```sql
 SHOW USER FUNCTIONS IN SCHEMA carto;
 SHOW USER PROCEDURES IN SCHEMA carto;
 ```
 
-##### 3. Create the functions and procedures
 
-Download the [modules script](https://storage.googleapis.com/carto-analytics-toolbox-core/snowflake/latest/sql/carto-analytics-toolbox-core-snowflake-modules.sql) into a local file.
+**Congratulations!** you have successfully installed the CARTO Analytics Toolbox in your Snowflake database. Now you can start [using the functions](/analytics-toolbox-snowflake/sql-reference/overview/). Please refer to Step 2 above to check the installed version and functions.
```

Comment:
I'd remove "Please refer to Step 2 above to check the installed version and functions."  (the "and functions" part is not correct now)

We could add the SELECT version to the script above or make the reference to the tip of step 2 before the congratulations:

> You can check the installed version as mentioned in Step 2